UnityProject/Assets/Scripts/Tilemaps/Behaviours/Meta/Atmospherics/ReactionManager.cs

```diff
@@ -73,58 +75,9 @@ private void Start()
 
 		private void Update()
 		{
-			#region wind
-
 			Profiler.BeginSample("Wind");
```

Comment:
was this supposed to be deleted after you're done profiling?

Reply:
well its a region for 3 lines so it seemed pointless, could add it back if you prefer

Reply:
no no, I mean the Profiler part